utils/config/daos_server.yml

```diff
@@ -169,6 +169,15 @@
 #helper_log_file: /tmp/daos_admin.log
 #
 #
+## Set the graceful shutdown timeout for DAOS I/O server termination.
+## After politely asking I/O server processes to exit when DAOS control
+## server is shutting down, when the timeout expires, the I/O server
+## processes are forcefully terminated.
+#
+## default: 10s
+#shutdown_timeout: 5s
```

Comment:
IMO this should be a hidden config value if we're keeping it. I.e. I wouldn't put it in an example config.